MediaBrowser.Api/Syncplay/SyncplayService.cs

```diff
@@ -160,7 +154,7 @@ public class SyncplayService : BaseApiService
         public void Post(SyncplayNewGroup request)
         {
             var currentSession = GetSession(_sessionContext);
-            _syncplayManager.NewGroup(currentSession);
+            _syncplayManager.NewGroup(currentSession, CancellationToken.None);
```

Comment:
I wonder if `request` has some cancellation token... if it doesn't let's leave it how it is now though.

Reply:
So that's where that token should come from. `request` doesn't have one unfortunately.